routers/router.php

```diff
@@ -41,6 +41,7 @@ abstract class Router {
   protected $config;
   protected $id;
   protected $requester;
+  protected $vrf_cmd;
```

Comment:
I'm not sure this has to be a property of all routers, maybe you can define it privately only for the nokia class.

Reply:
the idea was to close issue #54, so it would make sense for other vendors too.
You need than only to adapt the vendor specific code for cisco, juniper, ... and just place
$config['routers']['router1']['vrf'] = '1234';
per router in the config.php